Source/WebInspectorUI/UserInterface/Views/ColorSwatchDisplay.css

```diff
@@ -0,0 +1,17 @@
+.swatch-display ul{
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    gap: 0.2em;
```

Comment:
honestly id kinda rather us use `px` here as that's something easier for my head to wrap around (not to mention we use `px` basically everywhere else)